check_logs.py

```diff
@@ -801,13 +801,25 @@ def parse_lsr_error_log(args, log_url, extra_fields={}):
                     "Ansible Version": error_item["ansible_version"],
                     "Task": error_item["task_name"],
                     "Task Path": error_item["task_path"],
-                    "Parents": error_item["parents"],
-                    "Url": log_url,
+                    "Log Url": log_url,
                     "Detail": error_item["message"],
                 }
             )
             if role:
                 error["Role"] = role
+            optional_fields = [
+                ("Parents", "parents"),
+                ("Stdout", "stdout"),
+                ("Stderr", "stderr"),
+                ("RC", "rc"),
+                ("Start", "start_time"),
+                ("End", "end_time"),
+                ("Host", "host"),
+            ]
+            for title, field in optional_fields:
+                value = error_item.get(field)
+                if value or value == 0:
```

Comment:
For what case you do `value == 0:` ?

Reply:
the result code of a command or shell - `rc` - is an `int` and can have a `0` value, which we want to keep and show - it would be helpful in debugging to know that a task reported an error, but the `rc` is `0`